capstone/src/main/java/com/google/univiz/api/dataviz/CollegeDataDatavizConverter.java

```diff
@@ -0,0 +1,26 @@
+package com.google.univiz.api.dataviz;
+
+import com.google.common.base.Converter;
+import com.google.univiz.CollegeData;
+
+final class CollegeDataDatavizConverter extends Converter<CollegeData, CollegeStats> {
```

Comment:
Here are some suggested names for this class:
CollegeDataToDataVizConverter
CollegeDataToCollegeStatsConverter
CollegeDataVizConverter
CollegeDataStatsConverter